build.gradle

```diff
@@ -1,121 +1,19 @@
-
-import java.time.Instant
-import java.time.LocalDate
-
 plugins {
-    id 'java'
-    id 'groovy'
-    id 'org.springframework.boot' version '2.2.6.RELEASE'
-    id 'io.spring.dependency-management' version '1.0.9.RELEASE'
-    id 'org.owasp.dependencycheck' version '5.3.2'
-    id 'com.github.ben-manes.versions' version '0.28.0'
-    id 'org.jetbrains.kotlin.jvm' version '1.3.71'
-    id 'se.patrikerdes.use-latest-versions' version '0.2.13'
-}
-
-repositories {
-    mavenLocal()
-    mavenCentral()
-}
-
-def isNonStable = { String version ->
-    def stableKeyword = ['RELEASE', 'FINAL', 'GA'].any { it -> version.toUpperCase().contains(it) }
-    def regex = /^[0-9,.v-]+(-r)?$/
-    return !stableKeyword && !(version ==~ regex)
-}
-
-dependencyUpdates {
-    rejectVersionIf {
-        isNonStable(it.candidate.version) && !isNonStable(it.currentVersion)
-    }
-}
-
-configurations {
-    agentDeps
-    agentDeps.transitive = false
-}
-
-compileKotlin {
-    kotlinOptions {
-        jvmTarget = '11'
-    }
-}
-compileTestKotlin {
-    kotlinOptions {
-        jvmTarget = '11'
-    }
-}
-
-ext {
-    ext {
-        drivers = [
-                'chromeHeadless'
-        ]
-    }
-}
-
-dependencyCheck {
-    failBuildOnCVSS = 5
-    suppressionFiles = ['dependency-check-suppress-spring.xml']
-    format = 'ALL'
-    analyzers {
-        assemblyEnabled = false
-    }
-}
-
-project.sourceCompatibility = 11
-project.targetCompatibility = 11
-
-group = 'uk.gov.justice.digital.hmpps'
-
-def today = Instant.now()
-def todaysDate = LocalDate.now().format('yyyy-MM-dd')
-version = System.getenv('BUILD_NUMBER') ?: todaysDate
-
-bootJar {
-    manifest {
-        attributes 'Implementation-Title': rootProject.name, 'Implementation-Version': rootProject.version
-    }
-}
-
-springBoot {
-    buildInfo {
-        properties {
-            artifact = rootProject.name
-            version = version
-            group = group
-            name = rootProject.name
-            time = today
-            additional = [
-                    by                   : System.properties['user.name'],
-                    operatingSystem      : "${System.properties['os.name']} (${System.properties['os.version']})",
-                    machine              : InetAddress.localHost.hostName
-            ]
-        }
-    }
-}
-
-dependencyManagement {
-    imports {
-        mavenBom(SpringBootPlugin.BOM_COORDINATES)
-    }
+    id "uk.gov.justice.hmpps.gradle-spring-boot" version "0.1.0"
 }
 
 configurations {
     compile.exclude  module: "tomcat-jdbc"
 }
 
-ext ['spring-security.version'] = '5.3.0.RELEASE' // Updated since spring-boot-starter-oauth2-resource-server-2.2.5.RELEASE only pulls in 5.2.2.RELEASE (still affected by CVE-2018-1258 though)
+ext ['spring-security.version'] = '5.3.1.RELEASE' // Updated since spring-boot-starter-oauth2-resource-server-2.2.5.RELEASE only pulls in 5.2.2.RELEASE (still affected by CVE-2018-1258 though)
```

Comment:
One for later, but as this is related to the Owasp dependency check mitigation I think this should have already been moved to the plugin...

Reply:
interesting, didn't realise that.  Will investigate in next PR :+1: